graylog2-server/src/main/java/org/graylog2/migrations/V20161125142400_EmailAlarmCallbackMigration.java

```diff
@@ -153,7 +127,21 @@ public void doRun() {
         final ConfigurationRequest configurationRequest = this.emailAlarmCallback.getRequestedConfiguration();
 
         return configurationRequest.getFields().entrySet()
-            .stream()
-            .collect(Collectors.toMap(Map.Entry::getKey, entry -> entry.getValue().getDefaultValue()));
+                .stream()
+                .collect(Collectors.toMap(Map.Entry::getKey, entry -> entry.getValue().getDefaultValue()));
+    }
+
+    @AutoValue
+    @JsonAutoDetect
+    public static abstract class MigrationCompleted {
```

Comment:
You moved the `EmailAlarmCallbackMigrationPeriodical` over to a migration and changed the "migration done" marker class from `EmailAlarmCallbackMigrated` to `MigrationCompleted`.

That means this migration will run again even though the `EmailAlarmCallbackMigrationPeriodical` has been run already.

Just want to make sure this is intentional and that this doesn't cause any problems. :smiley: